tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md

```diff
@@ -15,6 +15,7 @@ Other free text and markdown formatting can be used elsewhere in the document if
 ### [Media Viewer preview not visible for files with .jpeg, .ogg, .webm and .gif formats](https://github.com/owncloud/ocis/issues/1490)
 -   [webUIPreview/imageMediaViewer.feature:141](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIPreview/imageMediaViewer.feature#L141)
 -   [webUIPreview/imageMediaViewer.feature:159](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIPreview/imageMediaViewer.feature#L159)
+-   [webUISharingInternalUsers/shareWithUsers.feature:317](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature#L317)
```

Comment:
Why is this being added here? @phil-davis

Reply:
@sakshamgurung please report the test output of this test and (if you know) mention why it starts to fail.

Then we can discuss and sort out what is the problem/difference with the later oCIS commit id. Maybe some regression will need to be fixed in oCIS, or web (real code or tests or...)

Reply:
@pascalwengerter The test fails to start due to https://github.com/owncloud/web/pull/6078#issuecomment-984327329 . And I added the scenario reference in the expected failure because this scenario was unable to show preview for the uploaded jpg image. I am pushing PR again.

Reply:
I don't get the relation here. Putting test cases into the expected failures of the acceptance tests doesn't keep the smoke tests from running/failing. Also I pointed out to @phil-davis and @individual-it that simply putting stuff into expected failures if they fail should not be our strategy of QA

Reply:
@sakshamgurung that comment is about an intermittent failure due to a startup timing problem of docker images. 95+% of the time that should not happen in CI. And when it does happen, the tests that fail will be "randomly" tests at the start of the pipeline that had the slow docker image pull.

You will not get reliable CI by adding a test to expected-failures because of this - 95+% of the time the test is going to pass!

Reply:
@phil-davis ji then I will remove my changes from expected-failure. I will try to push this PR later. I was confused with all these random fails.

Reply:
> Also I pointed out to @phil-davis and @individual-it that simply putting stuff into expected failures if they fail should not be our strategy of QA

Note: that is definitely not our strategy! If an existing test starts to fail then we must investigate why it started to fail. If it is a test code problem then we fix the test code. If there really is a regression in the system-under-test then we report that to the dev team. 

If the regression in the system-under-test is easy to fix, then we wait for the fix and then do the test commit id bump... to include the fixed code.

If the regression in the system-under-test will take significant time to fix, then we discuss this with the dev team, and a decision is made - either we wait for the fix, or we do add the failing tests to expected-failures with a link to the issue about the regression - but that must be sorted out in discussion with the development team.